### ISSUE TITLE
[docs] Elasticsearch settings missing in docs

### ISSUE THREAD

**Issue by crisdarocha** [elastic/elasticsearch#41702]
**Describe the feature**:

Settings like `"xpack.watcher.internal.ops.index.default_timeout"` are missing in our documentation.

For example, this specific setting, in it's old version (`"watcher.internal.ops.index.default_timeout"`) can be found in this older page (https://www.elastic.co/guide/en/watcher/current/configuring-default-internal-ops-timeouts.html, which you can reach coming from the general page that explains the settings (https://www.elastic.co/guide/en/watcher/current/actions.html#actions-index).

The new version (with `"xpack."`) doesn't have a counter part in the docs.

The general pages about the settings (https://www.elastic.co/guide/en/elastic-stack-overview/current/actions-index.html describe the feature, but doesn't tell you the setting to put in `elasticsearch.yml` to actually use this feature.

Also, unfortunately, not an isolated issue.

Thanks a lot.

**Elasticsearch version** : > 5.0

**Comment by lcawl** [comment]
If the Watcher folks agree, I'm fine with adding it to https://www.elastic.co/guide/en/elasticsearch/reference/master/notification-settings.html

**Comment by jakelandis** [comment]
I don't think we want to document the `.internal.ops` since they are not likely what the user wants to configure. 

For example, the index actions are controlled by
```
xpack.watcher.actions.index.default_timeout: 30s
xpack.watcher.actions.bulk.default_timeout: 120s
```
and are likely the timeouts the user cares about. For contrast, `xpack.watcher.internal.ops.index.default_timeout` only controls the timeout to update the watch status itself (not the index action of the watch). 

Even those ^^ I am on the fence about updating since the index action itself has a single `timeout` value (that is documented) which will override both defaults.

**Comment by crisdarocha** [comment]
@jakelandis if the omission of the setting is intentional, no problems with it. 

The situation get's complicated when the user comes mentioning having found a set of settings from version 2.4, which have "no counterpart" in the current versions.

But I see the point where `.internal.ops` settings are not openly advertised anymore.
